baselines/baseline_template/baseline_template/server.py

```diff
@@ -0,0 +1,6 @@
+##############################################################################
+# Please use this script to define the function called by your strategy in   #
```

Comment:
Let's use "module" in all the comments. A script would imply that this `.py` can be run, whereas a module implies that it's a `.py` files that just contains Python functions and classes.

```suggestion
# Please use this module to define the function called by your strategy in   #
```